tfhe/src/integer/server_key/radix_parallel/add.rs

```diff
@@ -1110,7 +1110,7 @@ impl ServerKey {
                 } else {
                     // u64::MAX is -1 in two's complement
                     // We apply the modulus including the padding bit
-                    u64::MAX % (1 << (block_modulus + 1))
+                    u64::MAX % (block_modulus * 2)
```

Comment:
does not look equivalent

Reply:
They are not equivalent, but the form (1 << (block_modulus + 1)) was incorrect